pycypher/LICENSE

```diff
@@ -0,0 +1,177 @@
+
```

Comment:
This is to include the license in built distributions (which apparently also contain source code), just to be safe.

Reply:
Ok. Perhaps we can copy it in at build time? I like to avoid the duplication if it's not necessary.

Reply:
It is not necessary but it definitely is not harmful here, the text of the license will not change. Secondly, it makes it more explicit, otherwise users may wonder whether the license from the root directory applies.
If you think that there is a good reason for adding yet another complexity to the build, I can change it so that it would be copied at build time.

Reply:
I just don't like the duplication. None of the other directories in the source tree contain a license file - it should be assumed that the project license applies to all the source.

Reply:
Ok, I will add copying the license to the build scripts.

Reply:
Done.